### ISSUE TITLE
Investigate disc speed

### ISSUE THREAD

**Issue by vadim2404** [neondatabase/autoscaling#329]
## Steps to reproduce
Performance is not comparable with pods

## Logs, links
https://neondb.slack.com/archives/C03TN5G758R/p1684147292620719

**Comment by cicdteam** [comment]
I measured disk performance in a simple pod and a simple virtual machine. Briefly - there is no difference in the speed of the disk.

### specs
#### Pod spec
```yaml
apiVersion: v1
kind: Pod
metadata:
  name: andrey-disk-speed
spec:
  nodeSelector:
    eks.amazonaws.com/nodegroup: main_m6i_metal
  terminationGracePeriodSeconds: 1
  containers:
  - image: debian:11
    imagePullPolicy: IfNotPresent
    name: tool
    args: ["/bin/sh", "-c", "sleep infinity"]
    resources:
      limits:
        cpu: "1"
        ephemeral-storage: 20Gi
        memory: 4Gi
      requests:
        cpu: "1"
        ephemeral-storage: 20Gi
        memory: 4Gi
```
#### VM spec
```yaml
apiVersion: vm.neon.tech/v1
kind: VirtualMachine
metadata:
  name: andrey-disk-speed
spec:
  restartPolicy: Never
  schedulerName: autoscale-scheduler
  guest:
    cpus:
      min: 1
      max: 1
      use: 1
    memorySlotSize: 1Gi
    memorySlots:
      min: 4
      max: 4
      use: 4
    rootDisk:
      image: neondatabase/vm-debian:11
      size: 20Gi
```

### Results
#### Used tool
Flexible I/O tester - [fio](https://fio.readthedocs.io/en/latest/index.html)
#### Random read-write
```console
Pod:

read:  IOPS=5076, BW=317MiB/s (333MB/s)(18.6GiB/60027msec)
write: IOPS=5078, BW=317MiB/s (333MB/s)(18.6GiB/60027msec); 0 zone resets

VM:

read:  IOPS=5079, BW=317MiB/s (333MB/s)(18.6GiB/60028msec)
write: IOPS=5081, BW=318MiB/s (333MB/s)(18.6GiB/60028msec); 0 zone resets
````
#### Sequential read
```console
Pod:

read: IOPS=16.4k, BW=1023MiB/s (1072MB/s)(59.9GiB/60019msec)

VM:

read: IOPS=16.4k, BW=1023MiB/s (1072MB/s)(59.9GiB/60017msec)
```

**Comment by sharnoff** [comment]
This was fixed with #509.
